omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -234,6 +234,20 @@
 default['private_chef']['opscode-erchef']['couchdb_max_conn'] = '100'
 default['private_chef']['opscode-erchef']['ibrowse_max_sessions'] = 256
 default['private_chef']['opscode-erchef']['ibrowse_max_pipeline_size'] = 1
+# general search settings used to set up chef_index
+default['private_chef']['opscode-erchef']['search_provider'] = "solr" # solr, TODO cloudsearch
+default['private_chef']['opscode-erchef']['search_queue_mode'] = "rabbitmq" # rabbitmq, batch, or inline
```

Comment:
We've talked about this a few time, but having 3 options makes me sad about the confusion that it's going to add. We're going to need to clearly document the differences between the three, and also the behavior of `batch`. A batch size of 5 MILLION (:drevilface:) is pretty insane as a default. What does that mean? What is the `10` for wait? 10 seconds or 10 milliseconds?